gammapy/modeling/models/temporal.py

```diff
@@ -334,6 +334,74 @@ def integral(self, t_min, t_max, **kwargs):
         return integral / self.time_sum(t_min, t_max)
 
 
+class GeneralizedGaussianTemporalModel(TemporalModel):
+    r"""A generalized Gaussian temporal profile
+
+    ..math::
+            F(t) = exp( - (\frac{|t - t_{ref}|}{\sigma_rise}) ^ nu)   for  t < t_ref
+            
+            F(t) = exp( - (\frac{|t - t_{ref}|}{\sigma_decay}) ^ nu)   for  t > t_ref            
+
+    Parameters
+    ----------
+    t_ref: `~astropy.units.Quantity`
+        The time of the pulse's maximum intensity.
+    sigma_rise : `~astropy.units.Quantity`
+        Rise time constant.
+    sigma_decay : `~astropy.units.Quantity`
+        Decay time constant.
+    nu : `~astropy.units.Quantity`
+        Pulse sharpness -> lower values implies a more peaked pulse
+    
+    """
+
+    tag = ["GeneralizedGaussianTemporalModel", "gengauss"]
+
+    _t_ref_default = Time("2000-01-01")
+    t_ref = Parameter("t_ref", _t_ref_default.mjd, unit = "day", frozen=False)
+    sigma_rise = Parameter("sigma_rise", "1d", frozen=False)
+    sigma_decay = Parameter("sigma_decay", "1d", frozen=False)
+    nu = Parameter("nu", 2, unit = "", frozen=False)
+
+    @staticmethod
+    def evaluate(time, t_ref, sigma_rise, nu, sigma_decay):
+        val_rise = np.exp( - (np.abs((time - t_ref).value) ** nu) / (sigma_rise.value ** nu))
+        val_decay = np.exp( - (np.abs((time - t_ref).value) ** nu) / (sigma_decay.value ** nu))
+        val = np.where(time < t_ref, val_rise, val_decay)
+        return val
```

Comment:
for consistency with the [GaussianTemporalModel](https://docs.gammapy.org/dev/_modules/gammapy/modeling/models/temporal.html#GaussianTemporalModel) a factor 2 could be added in the denominator within the exponential, so it would give the same result for equivalent parameters (i.e.  sigma=sigma_rise =sigma_decay and nu=2).